### PR TITLE
imgp: add page

### PR DESCRIPTION
**Pull request by 5shekel**
examples from project github
https://github.com/jarun/imgp/

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/imgp.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# imgp
+
+> imgp is a command line image resizer and rotator for JPEG and PNG images.
```
Comment (pxgamer) [inline pages/linux/imgp.md]: Change this to something like:
> A command line image resizer and rotator for jpeg and png images.

It's implied that this is describing `imgp`.

File: pages/linux/imgp.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+`imgp -x 1366x1000 ~/downloads ~/specifc.png`
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x 75 -w ~/image.jpg`
+
+- Rotate an image clockwise by 90 degrees:
+
+`imgp -o 90  ~/image.jpg`
+
```
Comment (pxgamer) [inline pages/linux/imgp.md]: Remove the trailing whitespace here (as per the linter). :+1:

File: pages/linux/imgp.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> imgp is a command line image resizer and rotator for JPEG and PNG images. 
+
+- Convert some images and directories:
+
+`imgp -x 1366x1000 ~/downloads ~/specifc.png`
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x 75 -w ~/image.jpg`
```
Comment (pxgamer) [inline pages/linux/imgp.md]: Same as the previous comment. :+1:

File: pages/linux/imgp.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# imgp
+
+> imgp is a command line image resizer and rotator for JPEG and PNG images. 
+
+- Convert some images and directories:
+
+`imgp -x 1366x1000 ~/downloads ~/specifc.png`
```
Comment (pxgamer) [inline pages/linux/imgp.md]: Could you put `1366x1000` in token syntax (i.e. `{{1366x1000}}`)?

Another thing, could you change the paths to use token syntax as well for each example?
`{{path/to/directory}}` and `{{path/to/file}}`

File: pages/linux/imgp.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x 75 -w ~/image.jpg`
+
+- Rotate an image clockwise by 90 degrees:
+
+`imgp -o 90  ~/image.jpg`
```
Comment (pxgamer) [inline pages/linux/imgp.md]: Same as the previous comment :+1:

File: pages/linux/imgp.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# imgp
+
+> Command line image resizer and rotator for JPEG and PNG images.
+
+- Convert some images and directories:
```
Comment (agnivade) [inline pages/linux/imgp.md]: We should clarify what does "convert" do ? Does it just resize to the target resolution ? And also what does "images and directories" mean ? In this context, it means we are converting directories too, which does not make sense.

File: pages/linux/imgp.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+`imgp -x {{1366x1000}} {{path/to/dir}} {{path/to/file}}`
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x 75 -w ~/image.jpg`
```
Comment (agnivade) [inline pages/linux/imgp.md]: Better to make it path/to/file to be consistent. Also 75 in tokens.

File: pages/linux/imgp.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+- Rotate an image clockwise by 90 degrees:
+
+`imgp -o 90  {{path/to/file}}`
```
Comment (agnivade) [inline pages/linux/imgp.md]: 90 in tokens.

File: pages/linux/imgp.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+# imgp
+
+> Command line image resizer and rotator for JPEG and PNG images.
+
+- Convert single images and/or whole directories containing valid image formats:
```
Comment (agnivade) [inline pages/linux/imgp.md]: End the sentence with `... to a target resolution:`

File: pages/linux/imgp.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+`imgp -x {{1366x1000}} {{path/to/dir}} {{path/to/file}}`
+
+- Scale an image by 75% and overwrite the source image:
+
+`imgp -x {{75}} -w ~/image.jpg`
```
Comment (agnivade) [inline pages/linux/imgp.md]: s/~/image.jpg/{{path/to/image}}

File: pages/linux/imgp.md (hunk 9 above)
Comment (5shekel) [reply pages/linux/imgp.md]: Thanks for keeping the high level for subbmision

File: pages/linux/imgp.md (hunk 10 above)
Comment (agnivade) [reply pages/linux/imgp.md]: Is there a reason you have changed this to path/to/directory without changing the description ? The description mentions "source image", hence I mentioned "path/to/image". I don't care about "image" or "directory". But the description and the example has to be consistent.

Comment (5shekel) [reply pages/linux/imgp.md]: my mistake

**Pull request merged**